alignak/objects/service.py

```diff
@@ -585,6 +585,18 @@ class Service(SchedulingItem):
 #                        |___/
 ######
 
+    def fill_predictive_missing_parameters(self):
+        """define state with initial_state
+
+        :return: None
+        """
+        if (self.initial_state == 'w'):
+            self.state = 'WARNING'
+        elif (self.initial_state == 'u'):
+            self.state = 'UNKNOWN'
+        elif (self.initial_state == 'c'):
+            self.state = 'CRITICAL'
```

Comment:
Hum for hot it's up/down/unreachable
see : https://github.com/Alignak-monitoring/alignak/blob/develop/alignak/objects/host.py#L1151-1160